### ISSUE TITLE
Java 6 compatibility

### ISSUE THREAD

**Issue by Verdoso** [aphyr/riemann-java-client#59]
Even though the pom.xml specifies Java 6 compatibility, the library is not Java 6 compatible as it uses core Java classes that are not present in Java 6, for example: java.lang.AutoCloseable.

**Comment by djarillo** [comment]
Also it uses java.util.concurrent.LinkedTransferQueue

**Comment by aphyr** [comment]
Yeah, Java 6 was EOLed several years ago; should probably flag everything as 7+.
